CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+**Features**:
+
+- Add dynamic `max_bytes`/`max_chars` derivation to `metastructure`. ([#5500](https://github.com/getsentry/relay/pull/5500))
```

Comment:
Seems more like an internal thing or no entry as there is no visible impact.

Reply:
Fair enough. I added one when I did this for PII but I guess that wasn't strictly necessary either.